components/schemas/booking-state

```diff
@@ -1,5 +1,6 @@
     booking-state:
       description: The life-cycle state of the booking (from NEW to FINISHED)
       type: string
-      enum: [NEW, BOOKED, CANCELLED, PAID, UPDATEREQUESTED, UPDATED, STARTED, FINISHED]
-      example: BOOKED
+      enum: [NEW, PENDING, RELEASED, EXPIRED, CONFIRMED, CANCELLED, STARTED, FINISHED]
```

Comment:
Can I suggest the term QUOTE instead of PENDING?  ?   “QUOTE” is a commonly used business term which exactly matches what you intend with the word PENDING…i.e. a one-sided commitment – the business is committed to providing it, but the recipient may decide not to take it.  “PENDING” implies that the recipient is expected to take it.